default-homescreens.json

```diff
@@ -12,6 +12,7 @@
     ], [
       ["outoftree_apps", "directory"],
       ["outoftree_apps", "sharing"],
+      ["outoftree_apps", "achievements"],
```

Comment:
I don't think that this should go here, but we're ditching this file soon anyways, so this is fine for now.